### PR TITLE
Fix /users tests

### LINKED ISSUE #10413: Make python3 fixes to "users" tests
https://travis-ci.org/mozilla/addons-server/jobs/480257962

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #10413 (I'll file another issue(s) for accounts and ratings.

### CODE REVIEW

File: src/olympia/addons/buttons.py [hunk 1]
```diff
@@ -17,20 +17,22 @@ def install_button(context, addon, version=None,
     """
     If version isn't given, we use the latest version.
     """
+    if not collection:
+        collection = None
```
Comment (eviljeff) [inline src/olympia/addons/buttons.py]: this was to fix a weird error where the undefined `collection` variable in the jinja2 template was undefined but not "undefined" enough to not break `hasattr`/`getattr`

File: .travis.yml [hunk 2]
```diff
@@ -21,12 +21,13 @@ jobs:
     - { stage: python3working, python: 3.6, env: TOXENV=codestyle}
     - { stage: python3working, python: 3.6, env: TOXENV=docs }
     - { stage: python3working, python: 3.6, env: TOXENV=assets }
+    - { stage: python3, python: 3.6, env: TOXENV=users }
```
Comment (diox) [inline .travis.yml]: You need to replicate that split to the python 2 jobs as well...

Comment (eviljeff) [reply .travis.yml]: https://github.com/mozilla/addons-server/pull/10450/commits/6ddf8f0753c14655f1ebc311104aa806fa150961

File: tox.ini [hunk 3]
```diff
@@ -49,10 +49,15 @@ commands =
     bash {toxinidir}/locale/compile-mo.sh {toxinidir}/locale/
     pytest -n 2 -m 'needs_locales_compilation' -v src/olympia/ {posargs}
 
-[testenv:users-accounts-and-ratings]
+[testenv:users]
```
Comment (diox) [inline tox.ini]: Just realized that you also need to update `envlist` at the top of this file.

Comment (eviljeff) [reply tox.ini]: https://github.com/mozilla/addons-server/pull/10450/commits/3cb082d8e15c25b1db7ec6be3658f70918626ed8

**Pull request merged**